packages/backend/src/rafiki/request/payment-pointer.request.ts

```diff
@@ -0,0 +1,44 @@
+import {CreatePaymentPointerMutationResponse, PaymentPointer} from "../generated/graphql";
+import {gql} from "graphql-request";
+import {requestGQL} from "../graphql.client";
+import env from "../../config/env";
+
+const OPEN_PAYMENTS_HOST = env.OPEN_PAYMENTS_HOST
+
+export async function createRafikiPaymentPointer(
+    accountName: string,
+    paymentPointerName: string,
+    assetId: string
+): Promise<PaymentPointer> {
+    const createPaymentPointerQuery= gql`
+    mutation CreatePaymentPointer($input: CreatePaymentPointerInput!) {
+      createPaymentPointer(input: $input) {
+        code
+        success
+        message
+        paymentPointer {
+          id
+          url
+          publicName
+        }
+      }
+    }
+  `
+    const createPaymentPointerInput = {
+        assetId: assetId,
+        url: `${OPEN_PAYMENTS_HOST}/${accountName}/${paymentPointerName}`,
+        publicName: paymentPointerName
+    }
```

Comment:
The account name is not unique at the moment. We can add a `unique` constraint, but I don't think it's a good idea. The users should be able to define their account name (Alice and Bob can both have an account named `USD Primary`). 

Having a check that verifies if the user already has an account with the given name would be okay (Bob should not be able to create two accounts with the name `USD Secondary`).

I think  we should drop `accountName` when creating a payment pointer and just rely on `paymentPointerName`.